test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -9,10 +9,22 @@
 public class NameTest {
 
     @Test
+    public void testConstructor_throwIllegalValueExceptionIfContainNonAlphabeticCharacters() {
```

Comment:
you can eliminate the word `test` from the method name; 
the test name should have 3 parts `<method under test>_<which scenario/test case>_<expected return value>`